README.md

```diff
@@ -1,16 +1,18 @@
-# Model Development Workflow Examples
+# Workflow Examples
 
-## Purpose
+**Purpose:**
 
-__These Tutorials use specific models to explain and demonstrate important steps of the DAGMC model workflow. By going through the tutorials one should understand all of the basic uses/functionality of SpaceClaim and Trelis/Cubit in the DAGMC workflow.__
+_These Tutorials use specific CAD models to explain and demonstrate important steps of the DAGMC workflow. By going through the tutorials one should understand the basic workflow of using DAGMC and the basic functionality of SpaceClaim and Trelis/Cubit in the DAGMC workflow._
 
-## Sections Sorted by Complexity of Model
-
-  * Overview of Tutorials:
+## Overview of Tutorials:
+  * Overview of tutorials that start with an existing CAD file:
```

Comment:
```suggestion
  * Tutorials that start with an existing CAD file:
```

Reply:
I will be implementing this and a few others that are similar.